lib/elixir/lib/inspect.ex

```diff
@@ -498,6 +454,32 @@ defimpl Inspect, for: Function do
     end
   end
 
+  def escape_name(name) when is_atom(name) do
+    escape_name(Atom.to_string(name))
+  end
+
+  def escape_name(string_name) when is_binary(string_name) do
+    case Macro.classify_identifier(string_name) do
+      :callable ->
+        string_name
+      type when type in [:not_callable, :alias] ->
+        "\"" <> string_name <> "\""
+      :other ->
+        Inspect.Atom.escape(string_name)
+    end
+  end
+
+  # Example of this format: -func/arity-fun-count-
+  def extract_nested_function_name_and_arity("-" <> rest) do
+    ["-", count, "-", "fun", "-", arity, "/" | reversed_function] =
+      rest
+      |> :re.split("([/-])")
```

Comment:
We can replace this call and next `Enum` call with `String.split(, ["/", "-"], trim: true)`.